README.md

```diff
@@ -315,6 +315,7 @@ readOnly | boolean | false | whether the document is read-only
 thumbnailViewEditingEnabled | boolean | true | whether use could modify through thumbnail view
 annotationAuthor | string | | the author name for all annotations in the current document
 continuousAnnotationEditing | boolean | false | whether annotations could be continuously edited
+tabTitle | String | document name | the tab title for the current document, if multiTabEnabled is true (this is currently `DocumentViewController` only for Android)
```

Comment:
Not sure what this Android-only thing is?

Reply:
I think it means for Android, it only works on widget ? Why only widget?

Reply:
Actually, `tabTitle` doesn't seem to work in the widget version on Android. Bug?